docs/core/run-time-config/threading.md

```diff
@@ -93,3 +93,39 @@ Project file:
 
 </Project>
 ```
+
+## `AutoreleasePool` for managed threads
+
+- Configures whether each managed thread receives an implicit [`NSAutoreleasePool`](https://developer.apple.com/documentation/foundation/nsautoreleasepool) when running on a supported macOS platform.
```

Comment:
Do we care about specifying that this is only on .NET 6+?

Reply:
I don't think that is needed because this is so tightly coupled to our Objective-C support. I can add it but I'm not sure if people will be confused.

Reply:
Yes, we've added it as another column in the table. For example: https://docs.microsoft.com/en-us/dotnet/core/run-time-config/garbage-collector#workstation-vs-server

Reply:
Thanks. Updated.